src/test/java/com/wikia/webdriver/testcases/mcfootertests/EnAnonMixedContentFooterTests.java

```diff
@@ -65,6 +69,7 @@ public void countNoOfArticlesInMCFooterWithDiscussionsAndWithMoreOfWikiArticles(
 
   @Test
   @Execute(onWikia = "glee")
```

Comment:
don't we want to change this wiki to some test one? since this wiki may not be accessible in staging environment